neo/Prompt/Commands/LoadSmartContract.py

```diff
@@ -64,6 +64,10 @@ def LoadContract(args):
     path = args[0]
     params = parse_param(args[1], ignore_int=True, prefer_hex=False)
 
+    for i in str(params):
+        if i == "f":
```

Comment:
I'm honestly surprised to see that this works. the `void` type is `0xFF` (or 255 in decimal) and not `f` (16 in decimal)